pkg/monitoring/rules/recordingrules/virt.go

```diff
@@ -43,7 +43,7 @@ func virtRecordingRules(namespace string) []operatorrules.RecordingRule {
 			},
 			MetricType: operatormetrics.GaugeType,
 			Expr: intstr.FromString(
-				fmt.Sprintf("sum(up{pod=~'virt-controller-.*', namespace='%s'}) or vector(0)", namespace),
+				fmt.Sprintf("sum(kube_pod_status_phase{pod=~'virt-controller-.*', namespace='%s', phase='Running'}) or vector(0)", namespace),
```

Comment:
`up` only refers to pods that are correctly reporting metrics, since this is a Prometheus metric
I think `kube_pod_status_phase` is much more relevant to know if a pod is up or not

and `correctly reporting metrics` seems more related and already tracked in `kubevirt_virt_controller_ready`